MineLeapMod/src/com/nautigsam/mineleapmod/ControllerSettings.java

```diff
@@ -155,18 +140,15 @@ public static LeapMotionMouse getLeapMotionMouse()
 		return LeapMotionMouse.getInstance();
 	}
 
-	public static void setInputEnabled(boolean b)
+	public static void setInputEnabled(boolean enable)
 	{
-		if (!isLeapMotionEnabled()) {

```

Comment:
This test is required, but it should not be with '!'. It prevents the Mouse to be instantiated before the native libraries are loaded.